tests/dotnet/UnitTests/TemplateTest.cs

```diff
@@ -41,6 +76,10 @@ public override string ToString ()
 			{
 				return Template;
```

Comment:
This controls how the test is rendered inside VSMac's test pad, so showing with the language makes it possible to distinguish the two `ios` templates:

```suggestion
				return TemplateWithLanguage;
```